crates/swc_ecma_parser/tests/tsc/parserRegularExpressionDivideAmbiguity3.ts

```diff
@@ -1 +1 @@
-if (1) /regexp/a.foo();
\ No newline at end of file
+if (1) /regexp/i.foo();
```

Comment:
Same here

Reply:
Test will fail. How should I do?

Reply:
I think we need something like https://github.com/swc-project/swc/blob/82069d180463193116ce2d1502934fa824ea8509/crates/swc/tests/tsc.rs#L16-L32 ?

(As these tests are copied from tsc periodically, it's better to ignore)